pkg/windows/windows.go

```diff
@@ -472,49 +474,26 @@ func (vm *windows) Bootstrap(desiredVer, apiServerURL string, credentials *Authe
 	if err := vm.transferFiles(); err != nil {
 		return errors.Wrap(err, "error transferring files to Windows VM")
 	}
-	if err := vm.configureContainerd(); err != nil {
-		return errors.Wrapf(err, "error configuring containerd")
-	}
-
-	if err := vm.ensureWICDSecretContent(credentials); err != nil {
-		return err
-	}
-	wicdBootstrapCmd := fmt.Sprintf("%s bootstrap --desired-version %s --api-server %s --sa-ca %s --sa-token %s",
-		wicdPath, desiredVer, apiServerURL, wicdCAFile, wicdTokenFile)
-	if out, err := vm.Run(wicdBootstrapCmd, true); err != nil {
-		vm.log.Info("failed to bootstrap node", "command", wicdBootstrapCmd, "output", out)
-		return err
-	}
-	return nil
-}
-
-// configureContainerd configures the Windows defender exclusion and starts the
-// Windows containerd service
-func (vm *windows) configureContainerd() error {
 	// if Windows Defender is running on the instance, create an exclusion for containerd
 	exclusionNeeded, err := vm.isWindowsDefenderRunning()
 	if err != nil {
 		return err
 	}
 	if exclusionNeeded {
-		if err := vm.createWindowsDefenderExclusion(containerdPath); err != nil {
+		if err := vm.createWindowsDefenderExclusion(ContainerdPath); err != nil {
```

Comment:
This is part of the containerd configuration and should be done as a powershell script as part of the containerd service definition in windows-services

Reply:
Good catch @sebsoto